misc/e2e-azure.sh

```diff
@@ -64,8 +64,6 @@ exec_on(){
 }
 
 stage-preflight() {
-  # TODO: move this to e2e-Dockerfile
-  pip install ansible -q -q -q --upgrade
   apt update && apt -y install
```

Comment:
oh what this `apt -y install` does?

Reply:
I was also wondering what the hell is that?

Reply:
git blame shows

```
-  apt update && apt -y install ed
+  apt update && apt -y install
```

we are not using `ed` now, so we can remove it :D